src/main/java/org/sagebionetworks/web/client/widget/entity/download/Uploader.java

```diff
@@ -790,7 +791,22 @@ public void onSuccess(Entity result) {
         entity = result;
         view.showInfo(DisplayConstants.TEXT_LINK_SUCCESS);
         if (successHandler != null) {
-          successHandler.onSuccessfulUpload();
+          synapseClient.getEntityBenefactorAcl(
```

Comment:
ideally this would be a jsClient call instead of a gwt rpc

Reply:
Agreed. I think we'll eventually be re-implementing this widget based on the other open tickets, so I went with the fastest option.